protos/orchestrator_service.proto

```diff
@@ -471,6 +471,15 @@ message PurgeInstancesResponse {
     google.protobuf.BoolValue isComplete = 2;
 }
 
+message RestartInstanceRequest {
+    string instanceId = 1;
+    bool restartWithNewInstanceId = 2;
```

Comment:
Can you explain the use case for this here? I have a concern this parameter is going to lead to ambiguity and customer confusion. Particularly, what does it do with the existing instance?

Does it terminate the existing instance? Or leave it alone?

Is there a way we can design this API to address that ambiguity?